File: java/estruturas-repetitivas/SomaImpares.java
import java.util.Scanner;

public class SomaImpares {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int x, y, aux, i, soma;

        soma = 0;

        System.out.println("Digite dois numeros: ");
        x = sc.nextInt();
        y = sc.nextInt();

        if (x > y) {
            aux = x;
            x = y;
            y = aux;
        }

        for(i = x+1; i < y; i++) {
            if(i % 2 != 0) {
                soma = soma + i;
            }
        }

        System.out.println("SOMA DOS IMPARES = " + soma);

        sc.close();
    }    
}

File: js/estruturas-repetitivas/somaImpares.js
const readline = require('readline-sync');

console.log("Digite dois numeros:");
let X = Number(readline.question());
let Y = Number(readline.question());

let soma = 0;

if (X > Y) {
   let aux = X;
   X = Y;
   Y = aux; 
}

for (let i = X+1; i < Y; i++) {
    if (i % 2 !== 0) {
        soma += i;
    }
}

console.log(`SOMA DOS IMPARES = ${soma}`);